### PR TITLE
Portenta Breakout datasheet: Add ethernet jumper pads info

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Add draftsman and section for the Ethernet jumper pads

### CODE REVIEW

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md [hunk 1]
```diff
@@ -340,7 +349,7 @@ Pins closer to the edge of the board are UART2. Pins closer to the center are UA
 
 ## Mechanical Information
 ### Board Outline
-![Board outline](assets/breakoutCarrierOutline.png)
+![alt_text](assets/breakoutCarrierOutline.png)
```
Comment (sebromero) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:352]: Image description is missing

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md [hunk 2]
```diff
@@ -126,8 +126,17 @@ The DIP switch allows for boot mode configuration:
 *   **BOOT SEL**: When set to ON, keeps the Portenta in Boot mode.
 *   **BOOT**: When set to ON enables the embedded bootloader. Firmware can be uploaded via the USB port on the breakout board (DFU). USB-A to USB-A  (non-crossover) cable required. The Portenta H7 has to be powered through the USB-C connector or VIN.
 
+### RJ-45 Connector
+The RJ-45 Connector allows to plug ethernet cables and connect to your net.
```
Comment (sebromero) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:130]:
```suggestion
The RJ-45 Connector allows to plug-in an ethernet cable and connect to your network.
```

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md [hunk 3]
```diff
@@ -78,8 +78,8 @@ This product is designed to work alongside the Portenta family. Please check the
 
 ## Functional Overview
 ### Board Topology 
-### Top View
-![Topology - Top view](assets/breakoutCarrierConnectors.png)
+Front view
+![](assets/breakoutCarrierConnectors.png)
```
Comment (sebromero) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:82]: Image description missing. (we may render datasheets as HTML on the website at some point)

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md [hunk 4]
```diff
@@ -90,8 +90,8 @@ This product is designed to work alongside the Portenta family. Please check the
 |   SW1    |               Boot mode selection                |   PB1    |       Power ON button       |
 |    U1    |               USBA power switch IC               |          |                             |
 
-### Bottom View
-![Topology - Bottom view](assets/breakoutCarrierConnectorsBottom.png)
+Back view
+![](assets/breakoutCarrierConnectorsBottom.png)
```
Comment (sebromero) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:94]: Image description missing

**Review by sebromero — DISMISSED** [review]
Found some minor issues

**Review by lunnea — APPROVED** [review]

**Comment by sebromero** [comment]
@marqdevx If you can add the image descriptions we can merge it

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md (hunk 1 above)
Comment (marqdevx) [reply content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md]: Fixed

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md (hunk 4 above)
Comment (marqdevx) [reply content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md]: Fixed

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md (hunk 3 above)
Comment (marqdevx) [reply content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md]: Fixed

**Pull request merged**